edx_sga/sga.py

```diff
@@ -250,6 +250,14 @@ def get_student_data():
                         'state': '{}',
                         'module_type': self.category,
                     })
+                if created:
+                   log.info(
```

Comment:
Unit test coverage shows this as missing, can you add a unit test to cover this, and also to verify that logging is happening?